ls1intum/Artemis
# `General`: Make course card clickable again

### Checklist
#### General


#### Client

### Motivation and Context


This PR fixes #4538 
### Description

As the missing link decreased the user experience, it is added again. There is some interference between link and the tooltips of the chart, so I increased the z-index of the chart. In order to make this part at least clickable as well, I handle the click via the angular event emitter. 
This solution is not optimal as some parts of the card are now clickable like a normal link (all three mouse buttons work) and the chart container is only reacting to left clicks. But this was the only solution I found so far. If you have an idea, feel free to contact me or leave feedback. 
### Steps for Testing

Prerequisites:
- 1 Student that is enrolled in at least one course

1. Log in to Artemis
2. Click on any part of the course card. Make sure the routing works and you are delegated to the course exercises page

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage





Only changes in HTML and SCSS files
### Screenshots


<details>
<summary>Current behavior</summary>

https://user-images.githubusercontent.com/80622272/148531795-cf1996a9-4957-41b7-baed-1f5afe68fadb.mp4


</details>
<details>
<summary>Changed behavior in this PR</summary>

https://user-images.githubusercontent.com/80622272/148531810-bb7a121c-0716-4878-b5cb-65e872f41fa3.mp4


</details>
